unity-renderer/Assets/Scripts/MainScripts/Debugging/WSS/WSSController.cs

```diff
@@ -398,7 +399,7 @@ void Update()
                                 CatalogController.i?.ClearWearableCatalog();
                                 break;
                             case "ConfigureHUDElement":
-                                HUDController.i?.ConfigureHUDElement(msg.payload);
+                                GetHUDBridge()?.ConfigureHUDElement(msg.payload);
```

Comment:
`hudBridge` local variable shall be initialized on `Awake` instead of being called each time. Wdyt? I'd look cleaner.

Reply:
The function will cache it. I copied the code from BiW.

```
        private BuilderInWorldBridge GetBuilderInWorldBridge()
        {
            if (builderInWorldBridge == null)
            {
                builderInWorldBridge = FindObjectOfType<BuilderInWorldBridge>();
            }

            return builderInWorldBridge;
        }

        private HUDBridge GetHUDBridge()
        {
            if (hudBridge == null)
            {
                hudBridge = FindObjectOfType<HUDBridge>();
            }

            return hudBridge;
        }
```

Let me know if you want me to change it anyway into the `Awake`